cmake/ctest/drivers/atdm/cts1/local-driver.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash -l
+
+if [ "${SALLOC_CTEST_TIME_LIMIT_MINUTES}" == "" ] ; then
+  SALLOC_CTEST_TIME_LIMIT_MINUTES=1:30:00
+  # This is just running tests, not the entire build!
+fi
+
+if [ "${Trilinos_CTEST_DO_ALL_AT_ONCE}" == "" ] ; then
+  export Trilinos_CTEST_DO_ALL_AT_ONCE=TRUE
+fi
+
+set -x
+
+source $WORKSPACE/Trilinos/cmake/ctest/drivers/atdm/ctest-s-driver-config-build.sh
+
+set -x
+
+atdm_run_script_on_compute_node \
+  $WORKSPACE/Trilinos/cmake/ctest/drivers/atdm/ctest-s-driver-test.sh \
```

Comment:
We should experiment with configuring, building, and running tests completely on the compute node.  Scot Swan says this works.